code_schemes/kakuma_s01e07_reasons.json

```diff
@@ -3,6 +3,86 @@
   "Name": "KAKUMA S01E07 Reasons",
   "Version": "0.0.0.1",
   "Codes": [
+    {
+      "CodeID": "code-6b83e95b",
+      "CodeType": "Normal",
+      "DisplayText": "culture or beliefs",
+      "NumericValue": 1,
+      "StringValue": "culture_or_beliefs",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-0d46de2f",
+      "CodeType": "Normal",
+      "DisplayText": "poverty",
```

Comment:
```suggestion
      "DisplayText": "poverty (financial reasons/source of income)",
```
?

Reply:
Merged to just poverty after a discussion with RDA.